src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -36,12 +128,30 @@
     }
 
     @Override
-    public int size() {
-        return 0;
+    public T get(int index) {
+        if (index < 0 || index >= size) {
+            throw new IndexOutOfBoundsException(
+                    "Index outside the list");
```

Comment:
Use a private `checkIndex` method to validate the index in the `get` method to avoid code duplication.